src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
```

Comment:
please also write colors in lowercase like #f3f3f3 , do the same changes in your variables.scss

p.s. I'm sorry I didn't notice this during the first check